common/CMakeLists.txt

```diff
@@ -1,766 +1,772 @@
-	repositories/base/base_character_corpses_repository.h
-	repositories/base/base_character_corpse_items_repository.h
-	repositories/base/base_character_currency_repository.h
-	repositories/base/base_character_data_repository.h
-	repositories/base/base_character_disciplines_repository.h
-	repositories/base/base_character_expedition_lockouts_repository.h
-	repositories/base/base_character_inspect_messages_repository.h
-	repositories/base/base_character_instance_safereturns_repository.h
-	repositories/base/base_character_item_recast_repository.h
-	repositories/base/base_character_languages_repository.h
-	repositories/base/base_character_leadership_abilities_repository.h
-	repositories/base/base_character_material_repository.h
-	repositories/base/base_character_memmed_spells_repository.h
-	repositories/base/base_character_pet_buffs_repository.h
-	repositories/base/base_character_pet_info_repository.h
-	repositories/base/base_character_pet_inventory_repository.h
-	repositories/base/base_character_potionbelt_repository.h
-	repositories/base/base_character_skills_repository.h
-	repositories/base/base_character_spells_repository.h
-	repositories/base/base_character_task_timers_repository.h
-	repositories/base/base_character_tasks_repository.h
-	repositories/base/base_char_create_combinations_repository.h
-	repositories/base/base_char_create_point_allocations_repository.h
-	repositories/base/base_char_recipe_list_repository.h
-	repositories/base/base_completed_tasks_repository.h
-	repositories/base/base_completed_shared_tasks_repository.h
-	repositories/base/base_completed_shared_task_members_repository.h
-	repositories/base/base_completed_shared_task_activity_state_repository.h
-	repositories/base/base_content_flags_repository.h
-	repositories/base/base_damageshieldtypes_repository.h
-	repositories/base/base_data_buckets_repository.h
-	repositories/base/base_db_str_repository.h
-	repositories/base/base_discovered_items_repository.h
-	repositories/base/base_doors_repository.h
-	repositories/base/base_dynamic_zones_repository.h
-	repositories/base/base_dynamic_zone_members_repository.h
-	repositories/base/base_dynamic_zone_templates_repository.h
-	repositories/base/base_eventlog_repository.h
-	repositories/base/base_expeditions_repository.h
-	repositories/base/base_expedition_lockouts_repository.h
-	repositories/base/base_faction_base_data_repository.h
-	repositories/base/base_faction_list_repository.h
-	repositories/base/base_faction_list_mod_repository.h
-	repositories/base/base_faction_values_repository.h
-	repositories/base/base_fishing_repository.h
-	repositories/base/base_forage_repository.h
-	repositories/base/base_friends_repository.h
-	repositories/base/base_global_loot_repository.h
-	repositories/base/base_gm_ips_repository.h
-	repositories/base/base_goallists_repository.h
-	repositories/base/base_graveyard_repository.h
-	repositories/base/base_grid_repository.h
-	repositories/base/base_grid_entries_repository.h
```

Comment:
Were these intentionally removed

Reply:
Specifically the grid ones

Reply:
The grid ones were removed because they are in the table exclude list in the generator because they had very early anti-patterns of our modern generator. I will actually modernize them so they show up here

Reply:
Pushed up